EventsExpress/ClientApp/src/config.js

```diff
@@ -1,4 +1,6 @@
 ﻿export default {
-  "FACEBOOK_CLIENT_ID": "2741444662848625",
-  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com"
+  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com",
+  "TWITTER_CALLBACK_URL": "https://eventsexpress-test.azurewebsites.net/Authentication/TwitterLogin",
```

Comment:
Do these changes work locally? Can you not use the relative url here?

Reply:
Reworked using the relative URL.